File: app/src/main/java/com/rR/apksapp/models/AppsApkDetails.java
package com.rR.apksapp.models;

import android.graphics.drawable.Drawable;

public class AppsApkDetails {
    private String appName;
    private String apkFilePath;
    private boolean isSelected;
    private Drawable drawable;

    public AppsApkDetails(String appName, String apkFilePath, Drawable drawable, boolean isSelected) {
        this.appName = appName;
        this.apkFilePath = apkFilePath;
        this.drawable = drawable;
        this.isSelected = isSelected;
    }

    public String getAppName() {
        return appName;
    }

    public String getApkFilePath() {
        return apkFilePath;
    }

    public Drawable getDrawable() {
        return drawable;
    }

    public void setIsSelected(boolean isSelected) {
        this.isSelected = isSelected;
    }

    public boolean getIsSelected() {
        return isSelected;
    }

}

File: app/src/main/java/android/rr/apksapp/models/AppsApkDetails.kt
package android.rr.apksapp.models

import android.graphics.drawable.Drawable

data class AppsApkDetails(val appName: String?, val apkFilePath: String?,
                          val drawable: Drawable?, var isSelected: Boolean) //{
    /*@JvmName("getAppName1")
    fun getAppName(): String? {
        return appName
    }

    @JvmName("getApkFilePath1")
    fun getApkFilePath(): String? {
        return apkFilePath
    }

    fun getDrawable(): Drawable? {
        return drawable
    }

    fun setIsSelected(isSelected: Boolean) {
        this.isSelected = isSelected
    }

    fun getIsSelected(): Boolean {
        return isSelected
    }*/
//}